<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseEditor</class>
 <widget class="QDialog" name="DatabaseEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Database Editor</string>
  </property>
  <widget class="QGroupBox" name="mangaInfoGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>Manga Info</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QHBoxLayout" name="mangaInfoTopLayout">
      <item>
       <layout class="QVBoxLayout" name="mangaInfoLeftLayout">
        <item>
         <widget class="QGraphicsView" name="mangaCoverGraphicView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>230</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>230</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mangaCoverAddButton">
          <property name="text">
           <string>Add Cover</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mangaGenresLabel">
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Genres:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="mangaGenresListWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="mangaGenresManageLayout">
          <item>
           <widget class="QComboBox" name="mangaGenresComboBox"/>
          </item>
          <item>
           <widget class="QToolButton" name="mangaGenreAddButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="mangaGenreRemoveButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="mangaPublicationDateLabel">
          <property name="text">
           <string>Publication Date:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="mangaPublicationDateEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="mangaPublicationStatusLabel">
          <property name="text">
           <string>Publication Status:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mangaPublicationStatusCombobox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="mangaInfoRigthLayout">
        <item>
         <widget class="QComboBox" name="mangaInfoAllComboBox"/>
        </item>
        <item>
         <widget class="QLabel" name="mangaTitleLabel">
          <property name="text">
           <string>Title:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mangaTitleLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="mangaDescriptionLabel">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="mangaDescriptionTextEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mangaAuthorsLabel">
          <property name="text">
           <string>Author(s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="mangaAuthorsListView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="mangaAuthorsManageLayout">
          <item>
           <widget class="QComboBox" name="mangaAuthorsComboBox"/>
          </item>
          <item>
           <widget class="QToolButton" name="mangaAuthorsAddButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="mangaAuthorsRemoveButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="mangaPublisherLabel">
          <property name="text">
           <string>Publisher:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mangaPublisherLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="mangaPublisherManageLayout">
          <item>
           <widget class="QComboBox" name="mangaPublishersComboBox"/>
          </item>
          <item>
           <widget class="QToolButton" name="mangaAddPublisherButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="mangaRemovePublisherButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mangaInfoButtonLayout">
      <item>
       <widget class="QPushButton" name="mangaInfoNewButton">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mangaInfoSaveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mangaInfoDeleteButton">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openGenreInfoEditButton">
        <property name="text">
         <string>Genre Info</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="authorInfoGroupBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>311</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Author Info</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QComboBox" name="authorsAllComboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="authorFullNameLabel">
      <property name="text">
       <string>Full Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="authorFullNameTextEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="authorCountryLabel">
      <property name="text">
       <string>Country of Birth:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="authorCountryTextEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="authorBirthdayLabel">
      <property name="text">
       <string>Birthday:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="authorBirthDateEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="authorWebsiteLabel">
      <property name="text">
       <string>Website:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="authorWebsiteTextEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="authorInfoButtonLayout">
      <item>
       <widget class="QPushButton" name="authorNewButton">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="authorSaveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="authorDeleteButton">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="publisherInfoGroubBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>311</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Publisher Info</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QComboBox" name="publishersAllComboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="publisherNameLabel">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="publisherNameTextEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="publisherCountryLabel">
      <property name="text">
       <string>Country:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="publisherCountryTextEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="publisherWebsiteLabel">
      <property name="text">
       <string>Website:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="publisherWebsiteTextEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="publisherNoteLabel">
      <property name="text">
       <string>Note:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="publisherNoteTextEdit">
      <property name="maximumSize">
       <size>
        <width>34243</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="publisherInfoButtonLayout">
      <item>
       <widget class="QPushButton" name="publisherNewButton">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="publisherSaveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="publisherDeleteButton">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
